third_party/bigframes_vendored/pandas/plotting/_core.py

```diff
@@ -6,8 +6,166 @@
 class PlotAccessor:
     """
     Make plots of Series or DataFrame with the `matplotlib` backend.
+
+    Parameters
+    ----------
+    data : Series or DataFrame
+        The object for which the method is called.
+    kind : str
+        The kind of plot to produce:
+
+        - 'line' : line plot (default)
+        - 'hist' : histogram
+        - 'area' : area plot
+        - 'scatter' : scatter plot (DataFrame only)
+    ax : matplotlib axes object, default None
+        An axes of the current figure.
+    subplots : bool or sequence of iterables, default False
+        Whether to group columns into subplots:
+
+        - ``False`` : No subplots will be used
+        - ``True`` : Make separate subplots for each column.
+        - sequence of iterables of column labels: Create a subplot for each
+          group of columns. For example `[('a', 'c'), ('b', 'd')]` will
+          create 2 subplots: one with columns 'a' and 'c', and one
+          with columns 'b' and 'd'. Remaining columns that aren't specified
+          will be plotted in additional subplots (one per column).
+
+          .. versionadded:: 1.5.0
```

Comment:
Given that we are extending pandas, shall we keep this note here?

Reply:
Good catch. Removed!